roles/romanesco-install/tasks/main.yml

```diff
@@ -0,0 +1,122 @@
+---
+
+  - name: apt | cache | update
+    apt: update_cache=yes
+    when: do_install|bool
+
+  - name: romanesco | deps | install
+    apt: name={{ item }} state=present update_cache=yes
+    with_items:
+      - apt-utils
+      - autoconf
+      - build-essential
+      - curl
+      - cmake
+      - g++
+      - gfortran
+      - git
+      - gzip
+      - libbz2-dev
+      - libcppunit-dev
+      - libexpat-dev
+      - libffi-dev
+      - libjpeg-dev
+      - libpng-dev
+      - libsqlite-dev
+      - libssl-dev
+      - libtool
+      - m4
+      - make
+      - mongodb
```

Comment:
I think we should remove mongodb from the requirements here; we already have another role for mongo I think, and romanesco doesn't depend on it strongly.